README.md

```diff
@@ -1,12 +1,5 @@
-# Google Software Product Sprint
+# My portfolio built during Google SPS
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains the web portfolio I worked on diring Google SPS program.
```

Comment:
Nit: Spelling.

Thanks for giving me a chance to leave a comment :)

Reply:
Fixed the spelling!